app/models/version.rb

```diff
@@ -332,8 +332,17 @@ def yanker
     Deletion.find_by(rubygem: rubygem.name, number: number, platform: platform)&.user unless indexed
   end
 
+  def prerelease
+    !!to_gem_version.prerelease?
+  end
+  alias prerelease? prerelease
```

Comment:
an inelegant fix, but the only sane one I could see until this is changed upstream in Rails at some future date.